docs/setup.rst

```diff
@@ -94,18 +95,97 @@ The following sub-sections provide guidance on how to install the recommended to
 - **Linux** TODO
 - **macOS** TODO
 
+.. _setup_zlib:
+
+zlib
+----
+:zlib:`zlib<>` is a common compression library.
+It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.
+
+- **Windows** On Windows, it's easiest to install using the packaged version available from :opencmiss_repo:`the OpenCMISS site<>` .
+  You're welcome to build your own version if you'd rather, but please make sure it's a 64-bit implementation.
+  To build :zlib:`zlib<>` from source refer to the next sub-section.
+- **Linux** zlib can be installed using :code:`sudo apt install zlib1g-dev`.
+- **macOS** zlib is already installed on macOS, so no further action is required on that platform.
+
+Building from source
+++++++++++++++++++++
+
+To create a zlib library suitable for libCellML we need to download, configure, build, and install it.
+
+To download zlib clone the repository https://github.com/OpenCMISS-Dependencies/zlib::
+
+  git clone https://github.com/OpenCMISS-Dependencies/zlib -b v1.2.3
+
+Create a build directory and change into it::
+
+  mkdir build-zlib
+  cd build-zlib
+
+Create the zlib library build instructions using CMake::
+
+  cmake -DCMAKE_INSTALL_PREFIX=<ZLIB_INSTALL_DIR> ..\zlib
+
+Where <ZLIB_INSTALL_DIR> is to be replaced by a path on your computer.  Something like::
```

Comment:
One sentence per line (makes maintenance easier). Also, wouldn't it be better to have <ZLIB_INSTALL_DIR> rendered as `<ZLIB_INSTALL_DIR>`?

Reply:
e1b924d